rtrlib/rtr/packets.c

```diff
@@ -369,6 +369,11 @@ static bool rtr_pdu_check_size (const struct pdu_header *pdu) {
     //TODO: Check the error msg 0 termination
 
     break;
+  case SERIAL_QUERY:
+  case RESET_QUERY:
+  case RESERVED:
+  default:
```

Comment:
I would argue that we should have all possible cases covered with the switch statement, even if not implemented yet. But it would be sensible to add some (debug) output that their implementation is missing. On the other hand: is it hard to implement those checks correctly?

If I get this right, they should be implemented anyway - so lets get this right now?